app/renderer/components/tabs/tab.js

```diff
@@ -333,20 +538,17 @@ class Tab extends React.Component {
         ref={(node) => { this.tabNode = node }}
         className={css(
           styles.tabArea__tab,
-
           // tab icon only (on pinned tab / small tab)
           this.props.isPinnedTab && styles.tabArea__tab_pinned,
           this.props.centralizeTabIcons && styles.tabArea__tab_centered,
           this.props.showAudioTopBorder && styles.tabArea__tab_audioTopBorder,
-
           // Windows specific style (color)
           isWindows && styles.tabArea__tab_forWindows,
-
           // Set background-color and color to active tab and private tab
           this.props.isActive && styles.tabArea__tab_active,
           this.props.isPrivateTab && styles.tabArea__tab_private,
           (this.props.isPrivateTab && this.props.isActive) && styles.tabArea__tab_private_active,
-
```

Comment:
nit: let's keep the blank lines above to make those comments meaningful :-)

Reply:
Whilst it highlights this specific area, it may harm the ability to scan of the rest of the render structure because typically blank lines should be used for component separation, and not usually in the middle of a component definition. I would suggest that the comments themselves act as padding between the lines of code, to highlight their importance or grouping. But this was your code so I’ll add the spacing back before I’m done. Thanks

Reply:
The reason why I thought the blanks lines are important is not only related with appearance but also   with style cascading; the sorting order in css() is so critical that reordering them without being sure how they are ordered would easily cause visual regressions (some of them would be critical) if the conditions are specified vaguely, while reordering components would basically not cause that kind of issue.

Ideally blanks lines inside css() would be removed, if the naming in BEM is properly done and makes it quite sure how the lines is aligned and why they are done so.

How does that sound? 😄